src/util/std_expr.h

```diff
@@ -691,6 +691,26 @@ class binary_exprt : public expr_protectedt
     check(expr, vm);
   }
 
+  exprt &lhs()
+  {
+    return exprt::op0();
+  }
+
+  const exprt &lhs() const
+  {
+    return exprt::op0();
+  }
+
+  exprt &rhs()
+  {
+    return exprt::op1();
+  }
+
+  const exprt &rhs() const
+  {
+    return exprt::op1();
+  }
+
   // make op0 and op1 public
   using exprt::op0;
```

Comment:
:pick: These are probably not required any more right? Should be marked as deprecated at least

Reply:
This is coming (they can only be marked deprecated once all uses are removed, or CI will fail).